daft/expressions/expressions.py

```diff
@@ -265,7 +265,7 @@ def __rmod__(self, other: Expression) -> Expression:
         return Expression._from_pyexpr(expr._expr % self._expr)
 
     def __and__(self, other: Expression) -> Expression:
-        """Takes the logical AND of two boolean expressions (``e1 & e2``)"""
+        """Takes the logical AND of two boolean or integer expressions (``e1 & e2``)"""
```

Comment:
Oh this should probably say something like `Takes the logical AND of two boolean expressions or bitwise AND of two integer expressions`. Same for `__or__` and `__xor__`.